packages/core/src/Stores/client-store.js

```diff
@@ -616,6 +620,8 @@ export default class ClientStore extends BaseStore {
         this.setIsLoggingIn(true);
         const authorize_response = await this.setUserLogin(login_new_user);
 
+        this.setDeviceData();
+
```

Comment:
Can we place this back into the `Redirect` component? Right now, `setDeviceData` will be called every time Client store is initialised even if there is no redirect param in the URL. (Or is that the intended use?)

Reply:
yes, that is intended, we want to setDeviceData everytime user come to deriv.app bringing the tracking query strings